cvat-canvas/src/typescript/canvasModel.ts

```diff
@@ -103,6 +103,7 @@ export interface BrushTool {
     color: string;
     form: 'circle' | 'square';
     size: number;
+    configureBlockedTools: (blockToolsConfiguration: Record<'eraser' | 'polygon-minus', boolean>) => void,
```

Comment:
Since it is a callback, I would suggest callback style naming. Maybe:

```suggestion
    onBlockUpdated: (blockedTools: Record<'eraser' | 'polygon-minus', boolean>) => void,
```

Reply:
Done